monero-wallet-gui.pro

```diff
@@ -214,6 +214,7 @@ linux {
         -lboost_program_options \
         -lssl \
         -lcrypto \
+        -ldl \
```

Comment:
IIRC this would kill the static build. So please add this on non static build only.